src/BMS.cpp

```diff
@@ -279,8 +279,8 @@ void BMS::clearVoltageReadings() {
     totalVoltage = 0;
 
     // Zero out each cell voltage
-    for (uint8_t i = 0; i < DEV::BQ76952::NUM_CELLS; i++) {
-        cellVoltage[i] = 0;
+    for (uint16_t& v : cellVoltage) {
+        v = 0;
```

Comment:
```suggestion
    memset(cellVoltage, 0, DEV::BQ76952::NUM_CELLS);
```
There are faster ways to zero out the cell voltages than using a for loop